include/oxen/quic/ip.hpp

```diff
@@ -5,6 +5,28 @@
 #include "formattable.hpp"
 #include "utils.hpp"
 
+/** IP Addressing Types:
+    - ipv{4,6} : These types represent raw ipv4/ipv6 addresses. Both hold their respective addresses in host order
+        internally, converting to network order when needed.
+    - ipv{4,6}_range : These types hold an extra uint8_t netmask, such that they can represent IP-ranges such as
+        1.2.3.4/16, 5.6.7.8/8, and so on. The underlying ipv{4,6} address is stored unmasked as a "networked IP address",
+        rather than an "IP network"
+    - ipv{4,6}_net : These types represent the above mentioned "IP network," storing the masked underlying ipv{4,6}
+        address.
+
+    Example:
+        ipv4 v4{"127.8.69.420"};
```

Comment:
This will throw an exception if you actually try it because 420 isn't a valid octet.